spec.emu

```diff
@@ -6,18 +6,89 @@
+<emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+<emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+<emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+<emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+<emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+<emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+
+<emu-clause id="sec-fundamental-objects">
+  <h1>Fundamental Objects</h1>
+
+  <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+  <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+  <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+
+  <emu-clause id="sec-symbol-objects">
+    <h1>Symbol Objects</h1>
+
+    <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+
+    <emu-clause id="sec-properties-of-the-symbol-constructor">
+      <h1>Properties of the Symbol Constructor</h1>
+
+      <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+      <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+      <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+      <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+
+      <emu-clause id="sec-symbol-isregistered">
+        <h1>Symbol.isRegistered ( _value_ )</h1>
+
+        <p>When the `isRegistered` method is called with arguments _value_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _result_ be Completion(thisSymbolValue(_value_)).
+          1. If _result_ is an abrupt completion, return *false*.
+          1. Assert: Type(! _result_) is Symbol.
+          1. For each element _e_ of the GlobalSymbolRegistry List (see 20.4.2.2), do
+            a. If SameValue(_e_.[[Symbol]], ! _result_) is true, return *true*.
+          1. Return *false*.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-symbol-iswellknown">
+        <h1>Symbol.isWellKnown ( _value_ )</h1>
+
+        <p>When the `isWellKnown` method is called with arguments _value_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _result_ be Completion(thisSymbolValue(_value_)).
+          1. If _result_ is an abrupt completion, return *false*.
+          1. Assert: Type(! _result_) is Symbol.
+          1. If ! _result_ is one of the values listed in <emu-xref href="#table-well-known-symbols"></emu-xref>, return *true*.
```

Comment:
[note] This will not include [%Intl%](https://tc39.es/ecma402/#intl-object).`[[FallbackSymbol]]` which was a point of discussion when we discussed Symbols As WeakMap keys.

Do we plan future editorial work on that table so it can reference 402? Do we plan to leave out that symbol out of this proposal?

I am fine merging as-is but an issue to decide on this should be opened after the merge.

Reply:
I believe the overwhelming preference is for FallbackSymbol to *not* be well-known, as the spec currently requires, and for v8 to fix it to not be cross-realm.

If it becomes well-known, 402 would need to add it to the table.

Reply:
Makes sense, still going to open the issue and put it in the slides, we can state our preference there!